wp-content/plugins/core/src/Templates/Controllers/Comments.php

```diff
@@ -1,11 +1,19 @@
 <?php
+declare( strict_types=1 );
 
+namespace Tribe\Project\Templates\Controllers;
 
-namespace Tribe\Project\Templates;
+use Tribe\Project\Templates\Abstract_Template;
+use Tribe\Project\Templates\Component_Factory;
+use Twig\Environment;
 
-use Tribe\Project\Twig\Twig_Template;
+class Comments extends Abstract_Template {
+	protected $path = 'comments.twig';
+
+	public function __construct( Environment $twig, Component_Factory $factory ) {
+		parent::__construct( $twig, $factory );
```

Comment:
Any particular reason we're leaving the constructor override in-place with only the parent call in here?

Reply:
Nope!